libs/blocks/marquee/marquee.css

```diff
@@ -548,12 +558,20 @@
     display: block;
   }
 
+  .marquee.large {
+    flex-direction: row;
+  }
+
+  .marquee.large.light .foreground {
+    background: unset;
+  }
+
   .marquee.large.desktop-dark {
     background-color: black;
   }
 
   .marquee.large.desktop-light {
-    background-color: initial;
+    background-color: blue;
```

Comment:
Is this intentional or forgotten after some testing? Just asking because we usually have variables for colors.

Reply:
Good catch, yes this was from a test.